flexget/plugins/sites/hliang.py

```diff
@@ -29,9 +29,15 @@ def parse_download_page(self, url, requests):
         txheaders = {'User-agent': 'Mozilla/4.0 (compatible; MSIE 5.5; Windows NT)'}
         try:
             page = requests.get(url, headers=txheaders)
+        except requests.exceptions.RequestException as e:
+            log.exception('Cannot open "%s"' % url, e)
```

Comment:
I'll change it, but can you explain why we should prefer log.error over log.exception?  I read the [documentation](https://docs.python.org/2/library/logging.html#logging.Logger.exception) that said log.exception is the same as log.error and should be called from the exception handler.

From [stackoverflow](https://stackoverflow.com/questions/5191830/best-way-to-log-a-python-exception#5191885) it seems that log.exception will automatically show the traceback and log.error won't, is this something we want in the log?